source/uninstall.sql

```diff
@@ -292,3 +294,5 @@ end;
 /
 
 spool off
+
+exit;
```

Comment:
new line missing

Reply:
Hi Jacek
I have applied fixes as suggested.
Attached also gifs screens for below query in jenkins cobertura plugin.
![jenkins_cobertura1](https://user-images.githubusercontent.com/7831112/36871765-2cf5d6fa-1d9b-11e8-9f7e-cf3c818449a9.gif)
![jenkins_cobertura2](https://user-images.githubusercontent.com/7831112/36871767-2d160682-1d9b-11e8-8bce-29d0931fdf17.gif)
![jenkins_cobertura3](https://user-images.githubusercontent.com/7831112/36871768-2d377772-1d9b-11e8-8a53-f6fa00f11852.gif)
![jenkins_cobertura4](https://user-images.githubusercontent.com/7831112/36871769-2d560c32-1d9b-11e8-871c-0553997af227.gif)




select * from
      table(
        ut3.ut.run(
          a_path => 'ut3',
          a_reporter=> ut3.ut_coverage_cobertura_reporter( ),
          a_source_files => ut3.ut_varchar2_list( 'test/sql_builder_pkg.pkb' ),
          a_test_files => ut3.ut_varchar2_list( )
        )
      );